.circleci/config.yml

```diff
@@ -28,6 +37,7 @@ jobs:
 workflows:
   circleci:
     jobs:
+      - lint
```

Comment:
We should add
```
filters:
  tags:
    only: /^\d+\.\d+$/
```
So this runs on releases as well (Counterintuitively, this means that circle will run this on tags, as by default it does not)